src/test/scala/mesosphere/marathon/core/instance/InstanceTestHelper.scala

```diff
@@ -0,0 +1,12 @@
+package mesosphere.marathon.core.instance
+
+import mesosphere.marathon.core.task.Task
+
+import scala.collection.immutable.Seq
+
+trait InstanceTestHelper {
```

Comment:
I think it's useful enough and may be useful in Instance anyways. If you don't agree, maybe follow the pattern common in scalatest "InstanceSupport" and force a self-type?